$NetBSD: patch-config.h.in,v 1.7 2019/05/01 17:59:56 maya Exp $

* define new path to if_tun.h.
* Revive tcp_wrappers support.

--- config.h.in.orig	2018-10-19 01:06:33.000000000 +0000
+++ config.h.in
@@ -910,6 +913,9 @@
 /* Define to 1 if you have the <net/route.h> header file. */
 #undef HAVE_NET_ROUTE_H
 
+/* Define to 1 if you have the <net/tun/if_tun.h> header file. */
+#undef HAVE_NET_TUN_IF_TUN_H
+
 /* Define if you are on NeXT */
 #undef HAVE_NEXT
 
@@ -1617,6 +1623,9 @@
 /* Define if pututxline updates lastlog too */
 #undef LASTLOG_WRITE_PUTUTXLINE
 
+/* Define if you want TCP Wrappers support */
+#undef LIBWRAP
+
 /* Define to whatever link() returns for "not supported" if it doesn't return
    EOPNOTSUPP. */
 #undef LINK_OPNOTSUPP_ERRNO
